### PR TITLE
update docker usage text for lfedge/eve

### PR DESCRIPTION
**Pull request by kgiori**
Signed-off-by: Kathy Giori <kathy.giori@gmail.com>

### CODE REVIEW

**Review by kgiori — COMMENTED** [review]
first commit had blank spaces at eol from copy/paste but second commit removed them

File: pkg/eve/runme.sh [hunk 1]
```diff
@@ -48,18 +48,49 @@ dump() {
...
-live and installer_raw support an optional last argument specifying the size of the image in Mb.
+Optionally you can pass arguments right after docker run:
+
+ -v <full path to local folder>:/in
+
+This allows you to overwrite files in the EVE-OS config partition with
+your own local modifications of those files (must have the same name).
+
+Example:
+docker run -v $HOME/eve/eve-overrides:/in --rm lfedge/eve:latest installer_raw > installer.raw
```
Comment (rvs) [inline pkg/eve/runme.sh:70]: Can we shorten it to just `$HOME/eve-overrides` ?

File: pkg/eve/runme.sh [hunk 2]
```diff
@@ -48,18 +48,49 @@ dump() {
...
+(Where your local "eve-overrides" directory contains one file "server"
+with one text string "some.eve-controller-url.com")
```
Comment (rvs) [inline pkg/eve/runme.sh:72]: I'd like this to read more as an example. E.g. `(Where your local "eve-overrides" directory may just contain...`

**Review by rvs — APPROVED** [review]
LGTM, with two small nits. Please take a look @kgiori

**Comment by kgiori** [comment]
@rvs good points, hopefully addressed your comments

**Comment by rvs** [comment]
Thanks @kgiori

**Pull request merged**